octoprint_mrbeam/static/js/app/view-models/modal/laserhead-changed.js

```diff
@@ -5,15 +5,79 @@ $(function () {
 
         self.settings = params[0];
         self.loginState = params[1];
+        self.laserhead_model_id = ko.observable();
```

Comment:
For JS the convention is to use camelCase, but I know we have all kinds of variable namings... :sweat_smile:

Reply:
I will change